pkg/ledger/volume_agg.go

```diff
@@ -0,0 +1,133 @@
+package ledger
+
+import (
+	"context"
+
+	"github.com/numary/ledger/pkg/core"
+	"github.com/numary/ledger/pkg/storage"
+)
+
+type TransactionVolumeAggregator struct {
```

Comment:
Why not just name it simply `transaction`?

Reply:
Less accurate?